### PR TITLE
PHR-13268  - Fix radio and checkbox

### PR DESCRIPTION
**Pull request by BalintViragJ**
[PHR-13268](https://pkbdev.atlassian.net/browse/PHR-13268) Fix radio and checkbox element bugs

[PHR-13268]: https://pkbdev.atlassian.net/browse/PHR-13268?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by dorogimate** [comment]
Commented on the JIRA ticket as well, tested manually and seems to be fine. Highlighted the parts that needs to be addressed. Comment I made on the ticket:
    1. Can add and remove checkboxes and radiobuttons correctly.
    2. Cursor doesn’t exit after each input
    3. Values are displayed correctly. **Do we need the value attribute for the input tags in the radio buttons?** (Doesn’t seem to effect the preview and output in any way though)
    4. **Despite the fact all the checkboxes and radio buttons have checked class name, they are not checked. Also in case of radio buttons there should be only one checked at a time. I think it would be good to remove all the checked class names.**

Was it on purpose that in case of checkboxes the remove button disappears if there is only one option left? It is good actually.

**Comment by BalintViragJ** [comment]
@dorogimate 

> Do we need the value attribute for the input tags in the radio buttons? 

It depends on whether they will be used in the future and if so how. I would rather keep tem.

> Despite the fact all the checkboxes and radio buttons have checked class name, they are not checked. Also in case of radio buttons there should be only one checked at a time. I think it would be good to remove all the checked class names.

thanks, removed the checked attributes from the class names

> Was it on purpose that in case of checkboxes the remove button disappears if there is only one option left? It is good actually.

yes, it was on purpose

File: src/app/elements/element-selection-tab/checkbox-element/checkbox-element-dialog/checkbox-element-dialog.component.ts [hunk 1]
```diff
@@ -11,8 +11,28 @@ import {DialogData} from "../../input-element/input-element-dialog/input-element
...
+      this.checkBoxValues[i] = this.bufferCheckBoxValues[i]
+    }
+  }
+
+  addCheckbox(){
+    this.checkBoxValues.push('VALUE' + (this.checkBoxValues.length + 1));
+  }
+
+  removeCheckbox(indexOfTheElement: number){
+    console.log(indexOfTheElement)
```
Comment (dorogimate) [inline src/app/elements/element-selection-tab/checkbox-element/checkbox-element-dialog/checkbox-element-dialog.component.ts:32]: nb: console log leftover

File: src/app/elements/element-selection-tab/checkbox-element/checkbox-element-dialog/checkbox-element-dialog.component.ts [hunk 2]
```diff
@@ -11,8 +11,28 @@ import {DialogData} from "../../input-element/input-element-dialog/input-element
...
+    if (this.checkBoxValues.length > 1) {
+      this.checkBoxValues.splice(indexOfTheElement,1);
```
Comment (dorogimate) [inline src/app/elements/element-selection-tab/checkbox-element/checkbox-element-dialog/checkbox-element-dialog.component.ts:34]: nb: space missing before `1`

File: src/app/elements/element-selection-tab/radio-element/radio-element-dialog/radio-element-dialog.component.ts [hunk 3]
```diff
@@ -11,10 +10,31 @@ import {Service} from "../../../../service/service";
...
+      this.radioElementValues[i] = this.bufferRadioElementValues[i]
+    }
+  }
+
+  addRadioButton(){
+    this.radioElementValues.push('VALUE' + (this.radioElementValues.length + 1));
+  }
+
+  removeRadioButton(indexOfTheElement: number){
+    console.log(indexOfTheElement)
```
Comment (dorogimate) [inline src/app/elements/element-selection-tab/radio-element/radio-element-dialog/radio-element-dialog.component.ts:33]: nb: logging leftover

**Review by dorogimate — APPROVED** [review]
LGTM

**Review by dorogimate — APPROVED** [review]

**Pull request merged**